routes/project/region.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+class Clover
+  hash_branch(:project_prefix, "region") do |r|
+    r.get true do
+      puts "project: #{@project.path}"
+      regions = @project.aws_location_credentials
+
+      if api?
+        Serializers::Region.serialize(regions)
+      else
+        @regions = Serializers::Region.serialize(regions)
```

Comment:
We should avoid the use of serializers as presenters for web views going forward.  Please change all views in this PR to call methods directly on the Region instances, and remove the use of serializers for web views.